templates/index.html

```diff
@@ -23,78 +29,129 @@
-            <li><i class="fas fa-code"> </i> Create amazing Github readme file for secret repository.Fork from here.<a href="https://github.com/suubh/suubh">View here</a></li>
+            <li>
+              <i class="fas fa-code"> </i> Horizontal and vertical navigation
+              using HTML and CSS.<a
+                href="https://suubh.github.io/CSS-Learning/Tables/verticalnav.html"
+                >View here</a
+              >
+            </li>
+            <li>
+              <i class="fas fa-code"> </i> Dropdown navigation using HTML and
+              CSS.<a
+                href="https://suubh.github.io/CSS-Learning/Tables/dropdown.html"
+                >View here</a
+              >
+            </li>
+            <li>
+              <i class="fas fa-code"> </i> Some assignment work.<a
+                href="https://suubh.github.io/CSS-Learning/Assignment/index.html"
+                >View here</a
+              >
+            </li>
+            <li>
+              <i class="fas fa-code"> </i> Create amazing Github readme file for
+              secret repository.Fork from here.<a
+                href="https://github.com/suubh/suubh"
+                >View here</a
+              >
+            </li>
           </ul>
         </div>
       </div>
-      
+
       <div class="contact-form">
         <h3 class="section-title">Contact us</h3>
         <p>Email (Required)</p>
-    
-        <input>
-        
+
+        <input />
+
         <p>Message (Required)</p>
-        
+
         <textarea></textarea>
-        
+
         <p>* Required fields must be filled</p>
-        
-        <input class="contact-submit" type="submit" value="Send">
-        
-        
+
+        <input class="contact-submit" type="submit" value="Send" />
       </div>
     </div>
 
     <div class="footer">
       <div class="footer-logo">Shubham Singh</div>
       <div class="footer-list">
         <ul>
-          <li><a href="https://www.facebook.com/profile.php?id=100007093204284"><i class="fab fa-facebook fa-2x"></i></a></li>
-          <li><a href="https://github.com/suubh"><i class="fab fa-github fa-2x"></i></a></li>
-          <li><a href="https://www.linkedin.com/in/shubham-singh-356ba5168"><i class="fab fa-linkedin fa-2x"></i></a></li>
+          <li>
+            <a href="https://www.facebook.com/profile.php?id=100007093204284"
+              ><i class="fab fa-facebook fa-2x"></i
+            ></a>
+          </li>
+          <li>
+            <a href="https://github.com/suubh"
+              ><i class="fab fa-github fa-2x"></i
+            ></a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/shubham-singh-356ba5168"
+              ><i class="fab fa-linkedin fa-2x"></i
+            ></a>
+          </li>
+          <li>
+            <i class="fas fa-code"> </i> This is the playground for the

```

Comment:
See this line. This is the code I added.
By far the concern of extending your code. It is due to the use of the prettier and eslint which is widely used by developed.
```suggestion
            <i class="fas fa-code"> </i> This is the playground for the

```